unpackaged/config/trial/profiles/Admin.profile

```diff
@@ -45,6 +45,9 @@
     <layoutAssignments>
         <layout>hed__Affiliation__c-%%%NAMESPACE%%%K12 Kit Affiliation Layout</layout>
     </layoutAssignments>
+    <layoutAssignments>
+        <layout>hed__Application__c-%%%NAMESPACE%%%K12 Kit Application Layout</layout>
+    </layoutAssignments>
```

Comment:
@lifewithryan @jofsky @jstvz Just wanted to know if this is the right place to add the Page layout for dev orgs? Coz this would assign it to Trial org. We would want it to work for both dev orgs and trial ors. FYI @kdy618

Reply:
@skosgi1 We currently use the trial org config for our dev and qa flows, so it will touch them all. Yes, we would want it to work for both.

Reply:
Thanks @MatthewBlanski